src/client/game.cpp

```diff
@@ -2977,11 +2982,11 @@ void Game::processPlayerInteraction(f32 dtime, bool show_hud, bool show_debug)
 	core::line3d<f32> shootline;
 
 	if (camera->getCameraMode() != CAMERA_MODE_THIRD_FRONT) {
-		shootline = core::line3d<f32>(camera_position,
-			camera_position + camera_direction * BS * d);
+		shootline = core::line3d<f32>(player_eye_position,
+			player_eye_position + camera_direction * BS * d);
 	} else {
 	    // prevent player pointing anything in front-view
```

Comment:
Eww space indention. As you are also adding a space in the line below, could you do
```suggestion
		// prevent player pointing anything in front-view
```
?

Reply:
Agreed.

Reply:
Sorry for late response. But, should I do or let other PR fixes this?

Reply:
I merged the PR because this request is somewhat trivial and the line was not touched in the patch. Create a PR if you want; not worth the effort IMO.

Reply:
I agree, it's not important to fix, but if you want to fix it i'll merge the PR immediately.

Reply:
I'll just leave it as it is.